sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/SqlParser.scala

```diff
@@ -292,6 +293,7 @@ class SqlParser extends AbstractSparkSQLParser with DataTypeParser {
     | AVG   ~ "(" ~> expression <~ ")" ^^ { case exp => Average(exp) }
     | MIN   ~ "(" ~> expression <~ ")" ^^ { case exp => Min(exp) }
     | MAX   ~ "(" ~> expression <~ ")" ^^ { case exp => Max(exp) }
+    | STDDEV  ~ "(" ~> expression <~ ")" ^^ { case exp => Stddev(exp)}
```

Comment:
We have changed how these plug in.  You'll need to change the [FunctionRegistry](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L82) now.